db/migrate/20250218070141_add_registered_at_timestamp.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AddRegisteredAtTimestamp < ActiveRecord::Migration[7.2]
+  def up
+    add_column :registrations, :registered_at, :datetime, precision: 6, default: -> { 'NOW(6)' }
```

Comment:
What happens when you just drop the `precision` key altogether?

Reply:
My understanding is that's equivalent to `precision: nil` - which means it defaults to MySQL's default, which is precision: 0 

[MySQL documentation](https://dev.mysql.com/doc/refman/8.4/en/date-and-time-type-syntax.html) supports this

![image](https://github.com/user-attachments/assets/929c172a-c03c-4cb3-880f-b7f7c3615e97)

Reply:
Okay, convinced